.travis.yml

```diff
@@ -1,14 +1,16 @@
 language: node_js
 
 node_js:
-- 4
+- 8
```

Comment:
I have always used Node 6 to run Babel, would it still work?

I think we should add something to the README, I don't see it mentioning which Node version must be used for building the module.

Reply:
I went directly to Node 8 with Travis for future-proofing, but Node 6 will definitely work. I will add a note to the documentation.